client/components/payment-activity/types.ts

```diff
@@ -1,4 +1,11 @@
 export interface DateRange {
+	key: string; // Key
 	date_start: string; // Start date
 	date_end: string; // End date
 }
+
+export interface DateRangeSelectorOptions {
+	key: string;
+	name: string;
+	hint: string;
+}
```

Comment:
Can we move these types closer to the place they are defined, e.g. in `DateRangePicker`. As I understand it, these types are part of the interface of the component. Keen for input from TypeScript expert @Jinksi (I defer to Eric here)

Is `DateRangeSelectorOptions` used?

Reply:
I agree, if possible we should move the TS interface within the file that it is intended for.

This matches our guidelines in [/docs/typescript/general-conventions.md](https://github.com/Automattic/woocommerce-payments/blob/8734-add-date-selector/docs/typescript/general-conventions.md#rely-on-type-inference):


> rely on inference and try to structure things in a way that empowers inference. Often, a few types go a long way.
>
> Try to keep the interfaces you do write close to their natural origin. For example, when you write a `Props` interface put it in the same module as the component.

Reply:
Addressed here - 8ca3357

`key` will be renamed after merging some other changes, to avoid conflicts.

Reply:
Changed here! [d909a63](https://github.com/Automattic/woocommerce-payments/pull/8927/commits/d909a63934d1e23fa51bbc2906b369bff6c0ecb7)